Covid19Radar/Covid19Radar/ViewModels/HomePage/NotifyOtherPageViewModel.cs

```diff
@@ -77,14 +75,17 @@ public NotifyOtherPageViewModel(INavigationService navigationService, UserDataSe
             {
                 var current = errorCount + 1;
                 var max = AppConstants.MaxErrorCount;
-                await UserDialogs.Instance.AlertAsync(AppResources.NotifyOtherPageDiag3Message,
-                    AppResources.NotifyOtherPageDiag3Title + $"{current}/{max}",
-                    Resources.AppResources.ButtonOk
-                    );
+                
+                string message = AppResources.NotifyOtherPageDiag3Title
+                    + $"({current}/{max})\n\n"
+                    + AppResources.NotifyOtherPageDiag3Message;
+                UserDialogs.Instance.ShowLoading(message);
                 await Task.Delay(errorCount * 5000);
+            } else
```

Comment:
[style]
改行が必要と思われます。　他所のelseを確認した所、ブランケットとelseは行を分けているようです。
```suggestion
            }
            else
```